app/middlewares/schema_validation.js

```diff
@@ -0,0 +1,8 @@
+const { checkSchema, validationResult } = require('express-validator/check');
+
+const checkValidationResult = (req, res, next) =>
+  validationResult(req).isEmpty() ? next() : next(validationResult(req).errors[0].msg);
```

Comment:
this is passing to the middleware error function all errors?

Reply:
Yes!

Reply:
can you show me an example of an error response when having more than one error?
seeing the errors[0] I think you are sending only one error message instead of all of them